packages/storybook-html/src/components/accordion/accordion.docs.mdx

```diff
@@ -0,0 +1,99 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+# Accordion
+
+<StatusBadge.Stable />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+Accordion component helps users to see only the content they need. Accordions expand and collapse sections of content.
+
+## Default example
+
+### a. Default
+
+<Canvas withSource="none">
+  <Story id="components-accordion--base" />
+</Canvas>
+
+```html
+<div class="accordion" aria-label="Accordion button group" role="presentation">
+    <div class="accordion__item">
+    <button class="accordion__toggle" aria-controls="content-1" aria-expanded="false" id="accordion-title-1">Step 1</button>
+    <div class="accordion__content" aria-hidden="true" id="content-1" aria-labelledby="accordion-title-1">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor eros dui, vitae iaculis leo ornare eget. Etiam tristique lobortis ligula non mattis. Ut dapibus libero neque, vel tincidunt nunc tempus eget.</p>
+    </div>
+    </div>
+    ...
+    ...
+</div>
+```
+
+### b. Default using list tags
```

Comment:
Drop Default

"Using list tags" 

This is a good example of where addtional information would be helpful. 

What are the list tags. provide a description and a code snipet. 
Why / when would you want to use list tags over the default option.

Reply:
Done